api/fftw3.h

```diff
@@ -50,16 +50,22 @@
 #include <stdio.h>
 
 #ifdef __cplusplus
+#  if __cplusplus >= 201103L && defined(FFTW_cpp_complex)
```

Comment:
Add these to cmake (and autotools)

Reply:
You wouldn't need them for compiling FFTW itself…

Reply:
Oh, good point. But still they need to be included in cmake export and pkg-config. Or at least documented somewhere

Reply:
They wouldn't be in pkg-config etcetera because you would need to enable this API in manually user code — it can't be turned on automatically, since that would break backwards compatibility.   Definitely would need docs, however.